test/position-managers/PositionManager.t.sol

```diff
@@ -199,11 +199,11 @@ contract PositionManagerTest is Test, PosmTestSetup, LiquidityFuzzers {
         uint256 balance1BeforeBurn = currency1.balanceOfSelf();
 
         decreaseLiquidity(tokenId, config, liquidity, ZERO_BYTES);
-        BalanceDelta deltaDecrease = snapLastDelta();
-        uint256 deltasSnapsLength = hook.getDeltasLength();
-        // No decrease/modifyLiq call will actually happen on the call to burn so the deltas array with be the same length.
+        BalanceDelta deltaDecrease = getLastDelta();
+        uint256 deltasSnapsLength = hook.numberDeltasReturned();
```

Comment:
oh sorry my comment was about `deltasSnapsLength` before - to remove the word snaps